docs/howto/writing-migrations.txt

```diff
@@ -183,3 +183,48 @@ the respective field according to your needs.
   Note there is a race condition if you allow objects to be created while this
   migration is running. Objects created after the ``AddField`` and before
   ``RunPython`` will have their original ``uuid``’s overwritten.
+
+.. _controlling_migration_order:
+
+Controlling the order of migrations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you auto-generate a migration using the :djadmin:`makemigrations`
```

Comment:
I think this paragraph doesn't need to emphasize makemigrations and auto-generated migrations so much. The concept is equally import when writing your own migrations from scratch.